interface/repository/pkmn_repository_test.go

```diff
@@ -0,0 +1,33 @@
+package repository
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/jvillarreal-w/academy-go-q42021/common"
+	"github.com/jvillarreal-w/academy-go-q42021/domain/model"
+	u "github.com/jvillarreal-w/academy-go-q42021/utils"
+)
+
```

Comment:
We usually like to use the assert package to compare the expected values with the ones our functions actually returned:
https://github.com/stretchr/testify